legend-engine-config/legend-engine-server/legend-engine-server-http-server/src/main/java/org/finos/legend/engine/server/core/pct/PCT_to_SimpleHTML.java

```diff
@@ -0,0 +1,256 @@
+// Copyright 2024 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.server.core.pct;
+
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.api.map.MutableMap;
+import org.eclipse.collections.api.multimap.list.ListMultimap;
+import org.eclipse.collections.api.multimap.list.MutableListMultimap;
+import org.eclipse.collections.api.tuple.Pair;
+import org.eclipse.collections.impl.factory.Lists;
+import org.eclipse.collections.impl.factory.Maps;
+import org.eclipse.collections.impl.utility.ListIterate;
+import org.finos.legend.pure.m3.pct.aggregate.generation.DocumentationGeneration;
+import org.finos.legend.pure.m3.pct.aggregate.model.Documentation;
+import org.finos.legend.pure.m3.pct.aggregate.model.FunctionDocumentation;
+import org.finos.legend.pure.m3.pct.functions.model.FunctionDefinition;
+import org.finos.legend.pure.m3.pct.functions.model.Signature;
+import org.finos.legend.pure.m3.pct.reports.model.AdapterKey;
+import org.finos.legend.pure.m3.pct.reports.model.FunctionTestResults;
+import org.finos.legend.pure.m3.pct.reports.model.TestInfo;
+import org.finos.legend.pure.m3.pct.shared.generation.Shared;
+import org.finos.legend.shared.stuctures.TreeNode;
+
+public class PCT_to_SimpleHTML
```

Comment:
Can we move this out of server module so we can re-use on VSCode LSP.  The server module carries too much.